### ISSUE TITLE
[Accessibility] List Card items are accessible via TAB key instead of arrow up/down keys only

### ISSUE THREAD

**Issue by yixuan-liu** [SAP/fundamental-ngx#8303]
Is this a bug, enhancement, or feature request?
--
A bug

Briefly describe your proposal
--
List items should be accessible via arrow up/down keys only

Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
--
Issue observed with Angular 13 + F-NGX 0.35.1

If this is a bug, please provide steps for reproducing it.
--
Focus onto the very first list item under this component, and then press TAB key will move on to the next list item

![Screen Shot 2022-06-30 at 12 31 57 PM](https://user-images.githubusercontent.com/22454655/176762054-0eead166-6669-421c-9923-30b8cd1cc757.png)

Is there anything else we should know?
-- 
Related SAP Ariba JIRA defect: https://product-jira.ariba.com/browse/SS-36895

**Comment by mikerodonnell89** [comment]
closing, appears to be fixed
